sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpConnectionScope.cs

```diff
@@ -341,6 +343,11 @@ protected AmqpConnectionScope()
                 var connection = await ActiveConnection.GetOrCreateAsync(timeout).ConfigureAwait(false);
                 cancellationToken.ThrowIfCancellationRequested<TaskCanceledException>();
 
+                if (string.IsNullOrEmpty(linkIdentifier))
```

Comment:
IsNullOrWhitespace?

Reply:
If someone is that intent on setting a blank identifier, they're welcome to do so.   We'll only auto-set when there's not one available.    _(I suppose it's fair to argue that "" is an explicit assignment too, but ...)_